lib/dolly/collection.rb

```diff
@@ -21,7 +23,7 @@ def single?
     private
 
     def collect_docs
-      lambda do |row|
+      proc do |row|
```

Comment:
On ruby 3.0 we cant call lambdas as proc arguments `any_method(&methd_returning_proc)`  wotn work if the method returns a `lambda`